DetectorDescription/DDCMS/interface/DDFilteredView.h

```diff
@@ -79,11 +79,16 @@ namespace cms {
     //! The full path to the current node
     const std::string path() const;
 
+    const std::vector<const Node*> geoHistory() const;
+
     //! The list of the volume copy numbers
     //  along the full path to the current node
     const std::vector<int> copyNos() const;
 
-    const std::vector<int> copyNumbers() { return copyNos(); }
+    template <typename... Ts>
```

Comment:
@ianna what is the reason of this update? My previous implementation was meant for compatibility, possibly usable in templated classes. Is this now breaking it? I am afraid I am not fully following the syntax, why to you need a templated method here, although with a not defined list of arguments?

Reply:
@ianna ok, but practically where is it used? I do not see in this PR a use case, please correct me if I am wrong. Do you have some further development in mind?

Reply:
https://cmssdt.cern.ch/lxr/search?%21v=CMSSW_11_1_X_2020-04-05-2300&_filestring=Geometry&_string=copyNumbers

Reply:
@ianna sure, but no template argument is passed as far as I can see. Anyway it is for my understanding, we may speak offline about this